nixos/modules/flyingcircus/roles/loghost.nix

```diff
@@ -0,0 +1,259 @@
+{ config, fclib, lib, pkgs, ... }: with lib;
```

Comment:
`with lib` is hard to spot while reading code -- better put it into a line of its own